.genesis/mcp/README.md

```
@@ -0,0 +1,197 @@
+```
+
+### `list-skills`
+List all available skills
+
+```typescript
+{
+  category?: string      // Optional filter
+}
+```
+
+### `list-instructions`
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)